<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>matchDialog</class>
 <widget class="QDialog" name="matchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>309</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Matching</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>511</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="102,100">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="15,72">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="30,35">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Sample Image:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selImgButton">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="PicWidget" name="picView" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="14,0,7,0">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Parameters</string>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>201</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You must choose &lt;span style=&quot; font-weight:600;&quot;&gt;at least one&lt;/span&gt;&lt;br&gt; item for equalization.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="RCheck">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Red</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="GCheck">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>100</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Green</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="BCheck">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>120</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Blue</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>160</y>
           <width>54</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Strategy</string>
         </property>
        </widget>
        <widget class="QComboBox" name="strategyBox">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>160</y>
           <width>121</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>SML</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GML</string>
          </property>
         </item>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,7,7">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="OKButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PicWidget</class>
   <extends>QWidget</extends>
   <header>picwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>OKButton</sender>
   <signal>clicked()</signal>
   <receiver>matchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>matchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selImgButton</sender>
   <signal>clicked()</signal>
   <receiver>matchDialog</receiver>
   <slot>openSample()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RCheck</sender>
   <signal>stateChanged(int)</signal>
   <receiver>matchDialog</receiver>
   <slot>modified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GCheck</sender>
   <signal>stateChanged(int)</signal>
   <receiver>matchDialog</receiver>
   <slot>modified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BCheck</sender>
   <signal>stateChanged(int)</signal>
   <receiver>matchDialog</receiver>
   <slot>modified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OKButton</sender>
   <signal>clicked()</signal>
   <receiver>matchDialog</receiver>
   <slot>applyMatch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openSample()</slot>
  <slot>modified()</slot>
  <slot>applyMatch()</slot>
 </slots>
</ui>
